src/common/server.ts

```diff
@@ -111,7 +111,11 @@ export async function restartServer(
   const lsClient = lsClientInstance.getLanguageClient();
   if (lsClient) {
     traceInfo(`Server: Stop requested`);
-    await lsClient.stop();
+    try {
+      await lsClient.stop();
+    } catch (e) {
+      traceInfo(`Server: Stop failed - ${e}`, '\nContinuing to attempt to start');
+    }
```

Comment:
I don't believe that is necessary in this case. The error received is directly from the python language server, which itself only sends a message. I do print the message in an output channel using `traceInfo`, but the most important part is that the error only occurs when the Server can not be stopped, which is because the server is not running.

The purpose of the function this code is in is to restart the language server, so in the event of an error stopping the server, we can just ignore it and continue to attempt to start the server again. I've logged everything I can via the `traceInfo` and the python server itself logs to the same output channel.